dashboard/app/models/plc/enrollment_module_assignment.rb

```diff
@@ -39,12 +39,12 @@ def status
   # in similar ways - look at all the levels, and see what the user progress is for them.
   def self.stages_based_status(stages, user, script)
     all_levels = stages.flat_map(&:script_levels).flat_map(&:levels)
-    levels_tracked = all_levels.reject {|level| [External, ExternalLink].include?(level.class) || level.try(:peer_reviewable) }
+    levels_tracked = all_levels.reject {|level| [External, ExternalLink].include?(level.class) || level.try(:peer_reviewable).try(:to_bool) }
 
     user_progress_on_tracked_levels = UserLevel.where(user: user, level: levels_tracked, script: script)
     passed_levels = user_progress_on_tracked_levels.passing
 
-    peer_reviewable_levels = all_levels.select {|level| level.try(:peer_reviewable)}
+    peer_reviewable_levels = all_levels.select {|level| level.try(:peer_reviewable).try(:to_bool)}
```

Comment:
For instance, you could add this to `free_response.rb`:
```ruby
  def peer_reviewable?
    self.peer_reviewable.try(:to_bool)
  end
```

and then here, just check `level.try(:peer_reviewable?)`